lib/views/booking_requests.erb

```diff
@@ -0,0 +1,32 @@
+<% if current_user != nil %>
+  <% @bookings.each do |mybookings| %>
```

Comment:
`mybookings` is not a great variable name. For one, it's not underscored. For two, it's pluralised. For three, if it's just a `Booking` instance, why not call it `booking`?